src/utils/common.ts

```diff
@@ -38,10 +38,25 @@ export const DEFAULT_SSO_ROUTES = {
     portal: 'https://console.stage.openshiftusgov.com',
   },
   frhStage: {
+    url: ['console.stage.openshiftusgov.com'],
+    sso: 'https://ocm-ra-stage-domain.auth-fips.us-gov-west-1.amazoncognito.com/login',
+    portal: 'https://console.stage.openshiftusgov.com',
+  },
+  ephem: {
     url: ['ephem.outsrights.cc'],
-    sso: 'https://stage-gov-console.auth.us-east-1.amazoncognito.com/login',
+    sso: 'https://keycloak-fips-test.apps.fips-key.2vn8.p1.openshiftapps.com',
     portal: 'https://ephem.outsrights.cc/',
   },
+  int: {
+    url: ['console.int.openshiftusgov.com'],
```

Comment:
Would it not be better to allow these URLs to be configured when building the assets via env variables and avoid exposing URLs like these on GitHub?

Reply:
@bastilian we configure the SSOUrl for our environments in the frontend operator and the values are not public. However, the specific environment of this PR is not yet using the frontend operator. It's in the process of migration. The same applies to the qaproduaht. Until qaprodauth is discontinued and all envs are using the FEO, we need these values still.